types/abi/src/standard.rs

```diff
@@ -205,7 +207,7 @@ pub fn get_call_wasm_contract_abi() -> SFXAbi {
         ingress_abi_descriptors: PerCodecAbiDescriptors {
             // assume all indexed in topics ("+")
             for_rlp: b"CallWasm:Log(caller+:Account32,contract+:Account32)".to_vec(),
-            for_scale: b"CallWasm:Enum(caller:Account32,contract:Account32)".to_vec(),
+            for_scale: b"Contracts:Struct(Call:Event(caller:Account32,contract:Account32))".to_vec(),
```

Comment:
Interesting, does this require that all targets must name their Pallet the same? This is interesting since some parachains do `EVM` (wrong as per rust style spec) and some do `Evm) for enum variants in  `construct_runtime!`

Reply:
hey, the naming is just a decoration used locally by ABI to access data per specific name - useful for SFX confirmation

Reply:
what's being checked is pallet + event index